NEMO/views/customization.py

```diff
@@ -259,6 +259,10 @@ class CalendarCustomization(CustomizationBase):
         "calendar_outage_recurrence_limit": "90",
         "calendar_qualified_tools": "",
         "calendar_configuration_in_reservations": "",
+        "create_reservation_confirmation": "",
+        "change_reservation_confirmation": "",
+        "reservation_confirmation_date_format": "MMMM D, yyyy",
+        "reservation_confirmation_time_format": "h:mma",
     }
 
```

Comment:
here, again if it's not too much trouble, I would wipe out user preferences that have the override on whenever the default is changed